Shopipy.ServiceManagement/Shopipy.ServiceManagement.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <OutputType>Library</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <RestorePackagesWithLockFile>true</RestorePackagesWithLockFile>
+        <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\Shopipy.Persistence\Shopipy.Persistence.csproj" />
+        <ProjectReference Include="..\Shopipy.CategoryManagement\Shopipy.CategoryManagement.csproj" />
```

Comment:
I would rather put Category management service interface to shared project than doing direct import